assets/main.scss

```diff
@@ -14,10 +14,27 @@
 		color: #e63;
 	}
 }
+
 .post-content h1 {
 	font-size: 42px;
 }
 
 .post-content hr {
 	margin: 1em auto;
 }
+
+p {
```

Comment:
I wonder if there is a way to programmatically enable/disable the styles...
I recommend: change this to `0.3em` or some such, to use relative sizing

Reply:
programmatically as in with a toggle? I'm sure we could, using html attributes -- see the CSS magic in #73